GettingStarted.md

```diff
@@ -131,28 +131,59 @@ You can run Kontain payload wrapped in a native Docker container.... in this cas
 
 #### Kontain OCI runtime (krun)
 
-**** KRUN is not in the bundle yet, skip this section ****
+You can use Kontain `krun` runtime from docker or podman directly. `krun` is installed together with KM and other components.
+`krun` is forked from Redhat's `crun` runtime github progrct, and can be invoked as `krun`.
 
-Or you can use Kontain `krun` runtime from docker/podman or directly. `krun` is installed together with KM and other components.
-`krun` is forked from Redhat's `crun` runtime, and can be invoked as `krun` or as `crun --kontain`
+An overview of container runtimes can be found here:
 
-Configuring `krun`: Documentation here is TBD
+https://medium.com/@avijitsarkar123/docker-and-oci-runtimes-a9c23a5646d6
 
-* Runtimes config
-*  OCI spec and use runtimes for bringing in existing kms. Update doc on how to use then and what needs to be in. Add an example
+Configuring `krun`:
 
-```
+* Runtimes config for docker
+
+Edit /etc/docker/daemon.json using sudo to run your editor and add the following:
+
+```txt
   {
-  "default-runtime": "runc",
-  "runtimes": {
-    "crun": {
-      "path": "/opt/kontain/bin/crun",
-      "runtimeArgs": [
-              "--kontain"
-      ]
+    "default-runtime": "runc",
+    "runtimes": {
+      "krun": {
+        "path": "/opt/kontain/bin/krun"
+      }
     }
   }
-  ```
+```
+
+Then restart docker for the change to take effect:
+
+```bash
+systemctl reload-or-restart docker.service
+```
+
+* Runtimes config for podman
+
+Edit /usr/share/containers/containers.conf using sudo to run your editor and add the following in the [engine.runtimes] section of the file:
+
+```txt
+  krun = [
+    "/opt/kontain/bin/krun",
+  ]
+```
+
+There is no need to restart podman.
+
+* krun package requirements
+
+krun needs the following packages to run:
+
+glibc
+yajl
+libseccomp
+libcap
+
+Install them with the appropriate package manager for your linux distribution.  Probably dnf or apt-get.
```

Comment:
pls add specific example for at least dnf

Reply:
Done.